### PR TITLE
mdd-added-what's-new

### PR DESCRIPTION
**Pull request by mddifilippo89**
mdd-added-what's-new

### CODE REVIEW

**Comment by mddifilippo89** [comment]
Waiting on final production versions to update {PackageVerChangedCharts}

File: docfx/en/components/toc.json [hunk 1]
```diff
@@ -27,7 +27,11 @@
     "status": ""
   },
   {
-    "exclude": ["Blazor"],
+    "name": "What's New",
+    "href": "whats-new-dv.md",
```
Comment (HUSSAR-mtrela) [inline docfx/en/components/toc.json:31]: add "general-" prefix

File: doc/en/components/whats-new-dv.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
```
Comment (HUSSAR-mtrela) [inline doc/en/components/whats-new-dv.md:1]: add "general-" prefix to file name

File: docfx/jp/components/toc.json [hunk 3]
```diff
@@ -27,7 +27,11 @@
     "status": ""
   },
   {
-    "exclude": ["Blazor"],
+    "name": "What's New",
+    "href": "whats-new-dv.md",
```
Comment (HUSSAR-mtrela) [inline docfx/jp/components/toc.json:31]: add "general-" prefix

File: docfx/kr/components/toc.json [hunk 4]
```diff
@@ -21,7 +21,11 @@
     "status": ""
   },
   {
-    "exclude": ["Blazor"],
+    "name": "What's New",
+    "href": "whats-new-dv.md",
```
Comment (HUSSAR-mtrela) [inline docfx/kr/components/toc.json:25]: add "general-" prefix

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**